### PR TITLE
Cryo no longer autoejects or shows reagents in the inserted beaker.

### PR DESCRIPTION
**Pull request by ghost**
I DON'T KNOW WHAT THE ACTUAL FUCK I'M DOING I THINK I BROKE IT BUT HOPEFULLY NOT
#### Changelog

:cl:
rscadd: Cryogenics machines have had the autoeject feature disabled, Nanotrasen accounting citing 'expenses' as the reason.
rescadd: Beakers no longer display their reagents inside the cryogenics tube.
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -76,17 +76,10 @@
...
-			playsound(src.loc, 'sound/machines/ding.ogg', volume, 1) // Bug the doctors.
-			if(autoeject) // Eject if configured.
-				open_machine()
-			return
-		else if(occupant.stat == DEAD) // We don't bother with dead people.
+		if(occupant.stat == DEAD) // We don't bother with dead people.
 			return
 
-		if(occupant.bodytemperature < T0C) // Sleepytime. Why? More cryo magic.
+		if(occupant.bodytemperature < T40C) // Sleepytime. Why? More cryo magic.
```
Comment (X-TheDark) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: And what would the purpose of this be?

**Comment by RoboticVodka** [comment]
I'm perfectly fine if you could upgrade the cryo to auto-eject. See? I care enough to discuss it on here aswell.

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 2]
```diff
@@ -145,9 +138,9 @@
 		return occupant
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/container_resist(mob/user)
-	user << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds.)</span>"
+	user << "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 2 minutes.)</span>"
 	audible_message("<span class='notice'>You hear a thump from [src].</span>")
-	if(do_after(user, 300))
+	if(do_after(user, 1200))
```
Comment (Time-Green) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:143]: This is about 3.5 minutes

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 3]
```diff
@@ -76,17 +76,10 @@
...
-			playsound(src.loc, 'sound/machines/ding.ogg', volume, 1) // Bug the doctors.
-			if(autoeject) // Eject if configured.
-				open_machine()
-			return
-		else if(occupant.stat == DEAD) // We don't bother with dead people.
+		if(occupant.stat == DEAD) // We don't bother with dead people.
 			return
 
-		if(occupant.bodytemperature < T0C) // Sleepytime. Why? More cryo magic.
+		if(occupant.bodytemperature < T0C + 40) // Sleepytime. Why? More cryo magic.
```
Comment (Time-Green) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:82]: Why did you up the temp for sleeping? This is unnecessary.

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 4]
```diff
@@ -11,7 +11,7 @@
 	var/volume = 100
 
 	var/efficiency = 1
-	var/sleep_factor = 750
+	var/sleep_factor = 1000
```
Comment (Time-Green) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:14]: ?

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 2 above)
Comment (ghost) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:143]: 2 minutes

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 3 above)
Comment (ghost) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:82]: sleep more, faster. needed.

Comment (ghost) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:82]: reason being it's too easy to be cryo'd for instant allheal of all damage which is a long known issue

a bunch of people attempt different solutions for it that fail-IE making cryo bad

the real solution is to go back to the way cryo was before autoeject feature was added-dangerous to use at the risk of someone killing you.

Get cryo'd, resist out to leave if a doctor forgets you. Being cryo'd could also lead to your death if a doctor uses it to murder you.

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 2 above)
Comment (Time-Green) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:143]: Err, still long

**Comment by BOBsHERE** [comment]
I approve of the cryo hiding its chemicals, disapprove of removing the eject button entirely. Make it a science upgrade.

**Comment by Cruix** [comment]
Let's try it and see how fast someone PRs to make the time to resist out shorter

**Comment by EMT321** [comment]
Two votes on the server showed uniform disapproval for this being merged. This needs more dialogue, and more time.

**Pull request merged**